### DISCUSSION TITLE
CSV Files

### DISCUSSION THREAD

**Discussion by SandeepRDiddi** [microsoft/FHIR-Converter#304] [General]
Need to know if we are receiving CSV Files from Source how can we use this Converter ? Can you provide some guidance

**Comment by ginalee-dotcom** [comment]
Hi, we have started working on JSON to FHIR conversion. Once this is available, you can easily convert CSV to JSON, then use our supported JSON to FHIR conversion tool.

**Reply by SandeepRDiddi** [reply]
Thanks for response @ginalee-dotcom : Do we know when can this be in Use ? up voted this request

**Comment by ejahnke** [comment]
I am also looking for this functionality! Any updates on the JSON conversion?
